File: DI/T1/Gestor/src/main/java/org/example/gestor/data/DataSet.java
package org.example.gestor.data;

import org.example.gestor.model.Usuario;

import java.util.ArrayList;

public class DataSet {
    public static ArrayList<Usuario> listaUsuarios = new ArrayList<>();

    public static void iniciarLista(){
        // String nombre, String correo, String profesion, String pass, String genero, String disponibilidad
        listaUsuarios.add(new Usuario("Borja","borja@gmail.com","profesor","123","Masculino","false"));
    }

    public static Usuario getUsuario(String correo, String pass){

        for (Usuario usuario: listaUsuarios) {
            if(usuario.getCorreo().equalsIgnoreCase(correo) && usuario.getPass().equalsIgnoreCase(pass)){
                return usuario;
            }
        }

        return null;
    }
}

File: PMDM/T2/Agenda/app/src/main/java/com/example/agenda/dataset/Dataset.kt
package com.example.agenda.dataset

import com.example.agenda.model.Usuario

class Dataset {

    companion object {

        val lista: ArrayList<Usuario> = arrayListOf(
            Usuario("borja@gmail.com", "1234"),
            Usuario("juan@gmail.com", "2345"),
            Usuario("luis@gmail.com", "3456"),
            Usuario("jose@gmail.com", "4567"),
        )


        fun comprobarLogin(correo: String, pass: String): Usuario? {
            return lista.find{ it.nombre.equals(correo) && it.pass.equals(pass) };
        }

        fun insertarUsuario(nombre: String, pass: String, perfil: String): Unit {
            lista.add(Usuario(nombre,pass,perfil))

        }
    }
}